### DISCUSSION TITLE
dailymotion.com: nuisance

### DISCUSSION THREAD

**Discussion by Pierre7855** [uBlockOrigin/uAssets#25007] [Q&A]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.lemonde.fr/
```

### Description

on this page, there is an embed video in autoplay mode. I would like to stop autoplay for this video.

### Other extensions used

no other extension.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yml
uBlock Origin: 1.59.0
Chromium: 128
filterset (summary):
 network: 182869
 cosmetic: 216981
 scriptlet: 36901
 html: 0
listset (total-discarded, last-updated):
 added:
  adguard-cookies: 31105-83, 21m
  ublock-cookies-adguard: 1605-1605, 21m
  fanboy-cookiemonster: 51702-4030, 6m
  ublock-cookies-easylist: 1606-9, 6m
  adguard-social: 23338-43, 21m
  fanboy-social: 16937-762, 6m
  fanboy-thirdparty_social: 68-0, 21m
  adguard-popup-overlays: 27860-1829, 6m
  adguard-mobile-app-banners: 5416-13, 6m
  adguard-other-annoyances: 14900-767, 6m
  adguard-widgets: 2640-33, 6m
  easylist-annoyances: 4725-78, 21m
  [4 lists not shown]: [too many]
 default:
  user-filters: 39-0, never
  ublock-filters: 39519-397, 2h.6m Δ
  ublock-badware: 10620-6, 2h.6m Δ
  ublock-privacy: 1068-2, 2h.6m Δ
  ublock-unbreak: 2505-0, 2h.6m Δ
  ublock-quick-fixes: 168-0, 2h.6m Δ
  easylist: 85392-108, 2h.6m Δ
  easyprivacy: 53057-22, 2h.6m Δ
  urlhaus-1: 29400-0, 6m
  plowe-0: 3547-993, 6m
  FRA-0: 25024-469, 6m
filterset (user): [array of 39 redacted]
userSettings:
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 215 ms (selfie)
 maxAssetCacheWait: 99 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 15
 network:
  lemonde.fr: 1
  amazon-adsystem.com: 1
  amplitude.com: 2
  appsflyer.com: 1
  batch.com: 1
  byspotify.com: 1
  chartbeat.com: 2
  ebxcdn.com: 1
  googlesyndication.com: 1
  keywee.co: 1
  outbrain.com: 1
  pbstck.com: 1
  pinimg.com: 1
 extended:
  ##.services
  ##.message--register
  ##.services:not(:has(div[data-services-name*="Le Monde"][data-se…
  ##.dfp-slot
  ###dfp-habillage
  ##.OUTBRAIN
  ##.footer__social
  ##.zone > div.area:has(> div.area__title > p:contains(Contenus p…
  ##body,html:style(height: auto !important; overflow: auto !impor…
```

</details>

**Comment by garry-ut99** [comment]
A duplicate of:
- https://github.com/uBlockOrigin/uBlock-issues/discussions/2942
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3098

Besides, I've tried this extension and it works:
- [AutoplayStopper](https://chromewebstore.google.com/detail/autoplaystopper/ejddcgojdblidajhngkogefpkknnebdh)

But if you're looking for a solution with uBlock:
- this solution works: https://github.com/gorhill/uBlock/wiki/Resources-Library#available-resources :
`||dailymotion.com$3p,frame,redirect=click2load.html,domain=lemonde.fr`
-  these solutions don't work in that case (but I still put it here so that you are aware of such functionalities): 
   - https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#permissions 
   `||lemonde.fr^$permissions=autoplay=()|unsized-media=()`
   `||dailymotion.com^$permissions=autoplay=()|unsized-media=()`
   - https://github.com/gorhill/uBlock/wiki/Per-site-switches#no-large-media-elements

<hr>

Tested videos:
- `https://www.lemonde.fr/international/video/2024/08/23/en-images-le-discours-de-kamala-harris-pour-clore-la-convention-democrate_6291494_3210.html`
- `https://www.lemonde.fr/international/video/2024/08/23/l-ukraine-detruit-des-ponts-en-russie-voici-pourquoi_6291701_3210.html
`

**Comment by Pierre7855** [comment]
Hello @garry-ut99 
You advised me the extension AutoPlayStopper in order to stop video autoplay, but I have just seen that it is working in Chrome with "manifest V2". As Google want to remove all "manifest V2" app, could you advise me an app working in "manifest V3" ?

**Reply by garry-ut99** [reply]
Because your provided info was: "uBlock Origin: 1.59.0" (MV2).
In case you want MV3 extension to stop autoplay, then unfortunately I didn't found any MV3 compatible so far.

But are you sure you want to switch to MV3 now, as uBO can still work one year after modifying browser's preferences:
- https://github.com/uBlockOrigin/uBOL-home/issues/170#issuecomment-2272920680

And after one year, you can switch to a Chromium fork, like Thorium or Supremium:
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3315

<hr>

Besides, the Google Chrome policy is: "Muted autoplay is always allowed.":
- https://developer.chrome.com/blog/autoplay?hl=en#new_behaviors

but I don't think their policy of muting video is sufficient, because:
- muted videos still waste internet bandwidth
- muted videos might be still distracting for users

Definately they should add an option to block videos from autoplay, just like Mozilla Firefox did: 
- https://support.mozilla.org/en-US/kb/block-autoplay
